app/src/main/java/com/alexP/socialnetwork/ui/screens/contacts/ContactsAdapter.kt

```diff
@@ -32,8 +35,16 @@ class ContactsAdapter(
             with(binding) {
                 textViewContactFullName.text = contact.fullName
                 textViewContactCareer.text = contact.career
-                imageViewContactImage.loadCircularImage(contact.photo)
-                buttonTrash.setOnClickListener {
+                if (contact.photo.isNotBlank()) {
+                    imageViewContactImage.loadCircularImage(contact.photo)
+                } else {
+                    imageViewContactImage.setImageResource(R.drawable.default_contact_image)
+                }
```

Comment:
I have not go deep in situation but in your code
```kotlin
fun ImageView.loadCircularImage(imageLink: String) {
    Glide.with(this).load(imageLink).apply(RequestOptions.circleCropTransform())
        .placeholder(R.drawable.default_contact_image).error(R.drawable.default_contact_image)
        .into(this)
}
```
does not placeholder() play role such as this if-else condition, does he?

Reply:
Yes, indeed